src/components/Editor/tests/Editor.spec.js

```diff
@@ -38,4 +38,20 @@ describe("Editor", () => {
     const { component } = render();
     expect(component).toMatchSnapshot();
   });
+
+  it("should set text", async () => {
+    const { component } = render();
+    component.setProps({ selectedSource: I.fromJS({ text: "test" }) });
+    // Todo: Add Checks for Set Text Working
+    expect(component).toMatchSnapshot();
+  });
+
+  it("should highlight line", async () => {
+    const { component } = render();
+    component.setProps({
+      selectedLocation: { line: 1, column: 1 },
+      selectedFrame: { location: { line: 1, column: 1 } }
+    });
+    expect(component).toMatchSnapshot();
```

Comment:
i don't think the snapshots are showing the highlight line or the text... perhaps we need to also spy on codemirror or do something to make sure they render...